### PR TITLE
Sidebar and tagingfor Technology and solution areas tagging/filtering

### PR DESCRIPTION
**Pull request by pravagar**
- Modified item.json, api controller to have Tachnologies instead of types. Renamed Category to solution areas.
- Removed runtime version as now we have Functions 1.x and functions 2.x as first class technologies.
- Modified item.json to have language and technologies string as to be displayed in UI.
- Modifed sidebar and tags component to include all technologies, solution areas and languages.
- Removed function app and logic app icon from tags

### CODE REVIEW

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 1]
```diff
@@ -491,12 +457,11 @@
     "template": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-transform-function/azuredeploy.json",
     "repository": "https://github.com/Azure/azure-quickstart-templates/tree/master/201-logic-app-transform-function",
     "description": "This template creates a webhook based C# azure function with transform capabilites to use in logic apps integration scenarios.",
-    "language": "csharp",
-    "type": "logicapp",
-    "category": [ "Integration", "Web API" ],
+    "language": "C# Script",
+    "technologies": [ "Logic Apps" ],
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: this should also include Function App in the technologies

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 2]
```diff
@@ -660,11 +615,10 @@
     "repository": "https://github.com/anirudhgarg/functions-logicapp-startstopvms",
     "description": "This solution allows you to Start and Stop Azure VM's for a given Subscription, Resource Group or Tags based on a schedule. The implementation uses Azure VM async API's and hence potentially hundreds of VM's can be started/stopped.",
     "language": "na",
-    "type": "logicapp",
-    "category": [ "Automation", "Integration", " Authentication" ],
+    "technologies": [ "Logic Apps" ],
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: This sample also has a 2.0 C# Function App

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 3]
```diff
@@ -950,38 +884,35 @@
     "template": "https://raw.githubusercontent.com/StratusOn/Media-Transport-to-UNC/master/src/Deployments/azuredeploy.json",
     "repository": "https://github.com/StratusOn/Media-Transport-to-UNC",
     "description": "Transports media files from a watched Azure Storage account to an Azure Files UNC file share.",
-    "language": "csharp",
-    "type": "functionapp",
-    "category": [ "Data Processing" ],
+    "language": "C#",
+    "technologies": [ "Functions 2.x" ],
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: why 2.x?

File: ServerlessLibraryAPI/wwwroot/items.json [hunk 4]
```diff
@@ -869,51 +809,47 @@
...
     "author": "priyaananthasankar",
-    "authortype": "Community",
-    "runtimeversion": "na"
+    "authortype": "Community"
   },
   {
     "id": "faae726b-1f09-4d1b-93d1-0240d0428a27",
     "createddate": "2019-02-26",
     "title": "Scheduled-based Python function for social media reporting",
     "repository": "https://github.com/gbowerman/twitter-code/tree/master/azurefunctions/dailyreport",
```
Comment (msnehagup) [inline ServerlessLibraryAPI/wwwroot/items.json]: this kind of repo url won't work for "Edit in vs code". We should update it

Comment (pravagar) [reply ServerlessLibraryAPI/wwwroot/items.json]: Yes. but in separate change

**Pull request merged**